guides/common/modules/proc_configuring-the-maximum-asynchronous-input-output-requests.adoc

```diff
@@ -1,7 +1,6 @@
 [id="Configuring_the_Maximum_Asynchronous_Input_Output_Requests_{context}"]
 = Configuring the Maximum Asynchronous Input-Output Requests
 
-In deployments using `katello-agent` infrastructure with a large number of content hosts, it may be necessary to increase the maximum allowable concurrent AIO requests.
 You can increase the maximum number of allowed concurrent AIO requests by increasing the kernel parameter `fs.aio-max-nr`.
```

Comment:
I now wonder if this is still a thing now that katello-agent is gone. @ehelms any idea?

Reply:
Based on the language of the line being removed I'd say this whole section can be dropped.